src/pages/en/install/auto.md

```diff
@@ -33,18 +35,60 @@ yarn create astro
+
+> Want to check out the templates in your browser before choosing? Visit [astro.new](https://astro.new/)
+
+### Install Dependencies (optional)
+The wizard will offer to run an `install` command for you at this time, which is optional.
+
+> If you do not do so at this time, you will need to [install dependencies](/en/install/auto#2-install-dependencies) after the wizard has finished, before starting your project.
+
+### Install any Official Astro Integrations (optional)
+You will be given the option at this time to add any [additional UI frameworks](/en/core-concepts/framework-components) (React, Svelte, Vue, Solid, Preact, Lit) and then other Astro official integrations (Tailwind, Turbolinks, Partytown, Sitemap) by running `astro add --yes`
+
+To select which Astro integrations, if any, you would like to include in your project, use the arrow keys (up and down) to navigate and the space bar to toggle selected/unselected. You can select multiple items at once, or you can continue without selecting any items.
+
+
+When you are satisfied with your selection, press return (enter) to submit. 
+
+>🟢 Selected integrations will display a solid (green) circle. (All integrations are **unselected by default**.) 
+>
+>⚠️ If every item is unselected, then no integrations will be added at this time. 
 
-## 2. Install Dependencies
 
-When the `create-astro` install wizard is complete, you should see some recommended instructions on your screen to follow that will help you complete setup and start your new project. 
+These integrations, and any [Astro community integrations](https://astro.build/integrations), can also be added later by following the instructions in our [integrations guide](/en/guides/integrations-guide).
 
-The only required step remaining is to install your project's dependencies using a package manager like npm:
+
+After selecting your integrations to add, you should see the following terminal message followed by **the entire contents of the `astro.config.mjs` file** that Astro will include with your project:
+
+```bash
+Astro will make the following changes to your config file:
+```
+
+This message should show that your chosen integrations have been successfully added to your project configuration. (If not, you can always add them manually later.)
+
+
+### Initialize a .git Repository (optional)
+
+At this final step, you can choose to initialize a git repository in your new directory. This is optional, but useful if you plan to use the tool [Git](https://git-scm.com/) for your project.
+
+### Next Steps
+
+When the `create-astro` install wizard is complete, you should see some recommended instructions ("Next Steps") on your screen to follow that will help you complete setup and start your new project.
+
+## 2. Install Dependencies
+ 
+If you did not install your project's dependencies using the wizard, you will now need to do so using a package manager like npm:
```

Comment:
```suggestion
If you did not install your project's dependencies using the wizard, you will now need to do so using your prefered package manager :
```

Reply:
☝️ type on "preferred" and stray space on "manager :"